pkg/models/upgrade_check.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+import "time"
+
+const (
+	UpgradeCheckInterval   = time.Second * 120
+	OperatorUpgradeChecker = "operatorUpgradeChecker"
+)
+
+type DockerTagsResponse struct {
```

Comment:
move it to `upgradechace` pkg

Reply:
I think it is better to keep structs in models pkg